types/vision/v4/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Ideally you would extend the default, I think we should lint this package given the changes are substential.